llvm/test/MC/Disassembler/X86/apx/ccmp.txt

```diff
@@ -0,0 +1,1598 @@
+# RUN: llvm-mc -triple x86_64 -disassemble %s | FileCheck %s --check-prefix=ATT
+# RUN: llvm-mc -triple x86_64 -disassemble -output-asm-variant=1 %s | FileCheck %s --check-prefix=INTEL
+
+## Condition flags
+
+# ATT:   ccmpoq {dfv=}	%rax, %rbx
+# INTEL: ccmpo {dfv=}	rbx, rax
+0x62,0xf4,0x84,0x00,0x39,0xc3
+
+# ATT:   ccmpoq {dfv=of}	%rax, %rbx
+# INTEL: ccmpo {dfv=of}	rbx, rax
+0x62,0xf4,0xc4,0x00,0x39,0xc3
+
+# ATT:   ccmpoq {dfv=sf}	%rax, %rbx
+# INTEL: ccmpo {dfv=sf}	rbx, rax
```

Comment:
The indentation before `{` is not unified. Below uses tab but here looks like single space. It think it's better to always use single space